src/main/resources/hudson/scm/SubversionTagAction/svn-tagform-update-row.js

```diff
@@ -0,0 +1,14 @@
+// update the visual feedback depending on the checkbox state
+function updateRow(changeEvent) {
+    const checkbox = changeEvent.target;
+    checkbox.parentNode.parentNode.style.color = checkbox.checked ? "inherit" : "grey";
+
+    const index = checkbox.getAttribute("data-index");
+    document.querySelector(`input[name="name${index}"]`).disabled = !checkbox.checked;
+}
+
+window.addEventListener("DOMContentLoaded", () => {
+    document.querySelectorAll(".svn-tagform-tag-checkbox").forEach(checkbox => {
```

Comment:
Elsewhere in Jenkins core and in other CSP PRs we're putting parameters in parentheses consistently, so please do so here as well:

```suggestion
    document.querySelectorAll(".svn-tagform-tag-checkbox").forEach((checkbox) => {
```

Reply:
Thanks, I'll try to be more consistent with it.